test/EFCore.MySql.Tests/ValueGeneration/Internal/MysqlSequentialGuidValueGeneratorTest.cs

```diff
@@ -0,0 +1,133 @@
+using System;
+using System.Linq;
+using Moq;
+using MySql.Data.MySqlClient;
+using Pomelo.EntityFrameworkCore.MySql.Internal;
+using Pomelo.EntityFrameworkCore.MySql.Storage.Internal;
+using Pomelo.EntityFrameworkCore.MySql.ValueGeneration.Internal;
+using Xunit;
+
+// ReSharper disable InconsistentNaming
+namespace Pomelo.EntityFrameworkCore.MySql.FunctionalTests.ValueGeneration.Internal
```

Comment:
The test class was moved to `EFCore.MySql.Tests`, so the namespace should be updated as well (see other test classes for reference).